src/deployable_example.cpp

```diff
@@ -737,32 +668,28 @@ void loop () {
     vTaskDelay(1000 / portTICK_PERIOD_MS);
   #endif
 
-  frame = esp_camera_fb_get();
-  // Testing how to get the latest image
-  esp_camera_fb_return(frame);
-  frame = esp_camera_fb_get();
```

Comment:
As far as I can tell, there isn't a clear reason to get and release before getting the frame we'll actually use. If there is, we can incorporate that into the frameLock class

Reply:
this may be to clear a buffered frame.  test with an actual camera and an image that is changing to confirm ?